### PR TITLE
Fixes #14077 - Add space before * in required fields

### PR DESCRIPTION
**Pull request by adamruzicka**

### CODE REVIEW

File: app/assets/stylesheets/bastion/forms.scss [hunk 1]
```diff
@@ -3,7 +3,7 @@
     color: inherit;
 
     &:after {
-      content: "*";
+      content: " *";
```
Comment (waldenraines) [inline app/assets/stylesheets/bastion/forms.scss]: While this likely works, I think we want to use this instead to be safe:

```css
  content: "\00a0 *";
```

Comment (adamruzicka) [inline app/assets/stylesheets/bastion/forms.scss]: Good idea, thanks

**Comment by waldenraines** [comment]
[test]

**Comment by waldenraines** [comment]
ACK, thanks @adamruzicka!

**Pull request merged**